doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -5,130 +5,148 @@ ACRN Configuration Tool
+      | **Native Linux requirement:**
+      | **Release:** Ubuntu 18.04+ or Clear Linux 30210+
+      | **Tools:** cpuid, rdmsr, lspci, dmidecode(optional)
+      | **Kernel cmdline:** "idle=nomwait intel_idle.max_cstate=0 intel_pstate=disable"
 
 #. Customize your needs.
 
    .. note:: **[TO BE DEVELOPED]**
-      The tool in this step is still under development, before its readiness user
-      could input the setting by editing the target XML file manually.
-
-   a. Copy ``$(BOARD).xml`` to host develop machine;
-   #. Run a UI based configure tool on host machine to input the expected scenario settings,
-      the tool will do sanity check on the input based on ``$(BOARD).xml`` and then generate
-      customized scenario based VM configurations in ``$(SCENARIO).xml``;
-   #. In the UI of configure tool, continue to input launch script parameter for
-      post-launched User VM. The tool will check the input based on both ``$(BOARD).xml``
-      and ``$(SCENARIO).xml`` and then generate launch script based VM configurations in
-      ``$(LAUNCH_PARAM).xml``;
+      The tool in this step is still under development. Until then,
+      you can input settings by editing the target XML file manually.
+
+   a. Copy ``$(BOARD).xml`` to the host develop machine.
+   #. Run a UI based configuration tool on the host machine to input the
+      desired scenario settings.  The tool will do a sanity check on the
+      input based on ``$(BOARD).xml`` and then generate a customized
+      scenario-based VM configurations in ``$(SCENARIO).xml``.
+   #. In the configuration tool UI, input the launch script parameters for the
+      post-launched User VM. The tool will sanity check the input based on
+      both ``$(BOARD).xml`` and ``$(SCENARIO).xml`` and then generate a launch
+      script-based VM configuration in ``$(LAUNCH_PARAM).xml``.
 
 #. Auto generate code.
-   There are three python tools will be used to generate configurations in patch format.
-   The patches will be applied to ``acrn-hypervisor`` git tree automatically.
 
-   a. Run ``python3 board_cfg_gen.py --board $(BOARD).xml`` under ``misc/board_config``
-      folder, it will generate patch for board related configurations;
-   #. **[TO BE DEVELOPED]** Run ``python3 scenario_cfg_gen.py --board $(BOARD).xml --scenario
-      $(SCENARIO).xml`` under ``misc/scenario_config`` folder, it will generate patch
-      for scenario based VM configurations;
-   #. **[TO BE DEVELOPED]** Run ``python3 launch_cfg_gen.py --board $(BOARD).xml
-      --scenario $(SCENARIO).xml --launch $(LAUNCH_PARAM).xml$`` under ``misc/launch_config``
-      folder, it will generate launch script for the specified post-launch User VM;
+   Python tools are used to generate configurations in patch format.
+   The patches will be applied to your local ``acrn-hypervisor`` git tree
+   automatically.
+
+   a. Generate a patch for the board-related configuration with::
+
+         cd misc/board_config
```

Comment:
@lirui34 Does this folder already exist, or must the user create it first?  (Same comment for the other two cases below).  Where is this misc folder?